Create blog-guidelines.md (#213)
ebpf-io/ebpf.io-website: blog-guidelines.md

* The most interesting posts are how-to blogs, blogs with technical content, and those that teach or show how to do something in a way others may not have thought of.
* Blog posts that show hurdles that were encountered and explain how they were overcome often do very well as the community is looking to learn.
* Content on how to choose between different technologies and how to accommodate different legacy issues and platforms is a good way to highlight your learnings and share with the community.
* When showing upstreaming of a patch fixing an issue for others, link back to the Github issue, so readers can follow along.
* Critical commentary or broad issues must be approached with sensitivity, professionalism, and tact in a way that is beneficial and positive for the community.

> * Your post must be your content. We are also happy to link to external blog posts if they are published elsewhere.

Author: ```suggestion
* Your post must be your content. Everything published on ebpf.io will be under the [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0/). Your post can be republished to another website under those terms.
* We are also happy to link to external blog posts if they are published elsewhere.
* Translations of blog post are also accepted.
```

Authors can add 1-2 sentences at the end of their blog with a link to an external site such as their website or a gated asset. Examples for best practices:
* Share contact info of  speaker - twitter, slack, email
* Links to company websites
* Links to projects websites or Github
* Can advertise if they are hiring and who to contact if interested

We will review the statement, as we do for the content for the blog, to approve the content. If changes are needed, we will provide suggested changes. 

## Promotion ##

> Please feel free to share the blog on Twitter and reach out to [blog@ebpf.io](blog@ebpf.io) if you would like to request ebpf.io's handle RTs.

Author: @ebpfsummit

> Please feel free to share the blog on Twitter and reach out to [blog@ebpf.io](blog@ebpf.io) if you would like to request ebpf.io's handle RTs.

Reviewer: OK, the phrasing was not clear to me. Maybe: “... if you would like to request retweets from @ebpfsummit” or “from the account associated to ebpf.io”.

# ebpf.io Blog Guidelines #

## ebpf.io Blog Overview ##

> The ebpf.io (CNCF) blog serves as a channel for eBPF project maintainers, contributors, and anyone interested in eBPF to share content with the eBPF community.

Reviewer 2: Why the `(CNCF)`? Might be bit confusing since it's not a CNCF project.

Reviewer 2: Otherwise lgtm.

Author: Seems I missed that part in the copy paste. I've removed now

> The ebpf.io (CNCF) blog serves as a channel for eBPF project maintainers, contributors, and anyone interested in eBPF to share content with the eBPF community.

Author: ```suggestion
The ebpf.io blog serves as a channel for eBPF project maintainers, contributors, and anyone interested in eBPF to share content with the eBPF community.
```

The content on our blog consists of:
* Articles about eBPF projects, whether on the eBPF Landscape or not
* Technical content and how-to’s
* Stories about eBPF project deployments
* Use cases and success stories
* Industry insight into eBPF adoption
* Reports from eBPF related events

Other topics are welcome, but it needs to have a direct link to the eBPF community. We accept content from anyone and external links are welcome.

## Content ##

> The CNCF blog audience is developers, IT operators, and eBPF and open source enthusiasts. Keep this in mind as you develop your blog content.

Reviewer 2: nit: CNCF -> ebpf.io

* The most interesting posts are how-to blogs, blogs with technical content, and those that teach or show how to do something in a way others may not have thought of.
* Blog posts that show hurdles that were encountered and explain how they were overcome often do very well as the community is looking to learn.
* Content on how to choose between different technologies and how to accommodate different legacy issues and platforms is a good way to highlight your learnings and share with the community.
* When showing upstreaming of a patch fixing an issue for others, link back to the GitHub issue (or upstream patch, mailing list discussion, etc.), so readers can follow along.
* Critical commentary or broad issues must be approached with sensitivity, professionalism, and tact in a way that is beneficial and positive for the community.
* Your post must be your content. Everything published on ebpf.io will be under the [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0/). Your post can be republished to another website under those terms.
* We are also happy to link to external blog posts if they are published elsewhere.

> * Translations of blog post are also accepted.

Reviewer: “a blog post” or “blog posts”?

> * Translations of blog post are also accepted.

Author: ```suggestion
* Translations of blog posts are also accepted.
```

Authors can add 1-2 sentences at the end of their blog with a link to an external site such as their website or a gated asset. Examples for best practices:
* Sharing contact info of the author: Twitter, Slack, email...
* Links to company websites
* Links to projects websites or GitHub
* Advertising if they are hiring and who to contact if interested

We will review the statement, as we do for the content for the blog. If changes are needed, we will provide suggestions.

## Promotion ##

> Please feel free to share the blog on Twitter and reach out to [blog@ebpf.io](blog@ebpf.io) if you would like to request ebpf.io's handle RTs.

Author: ```suggestion
Please feel free to share the blog on Twitter and reach out to [blog@ebpf.io](blog@ebpf.io) if you would like to request retweets from [@ebpfsummit](https://twitter.com/eBPFsummit).
```
